### DISCUSSION TITLE
[New Feature] bullet/projectile tracer

### DISCUSSION THREAD

**Discussion by XNiPERIANZ** [Vinifera-Developers/Vinifera#1267] [Feature Requests]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, videos, links, etc)*.

### Description

currently projectile or bullet weapontype without using special tag like [ROT=1] can only travel below 5-6 cell even the unit have the same height as the target, it looks ridiculous when you trying to make unit based on energy weapon, like plasma projectile, still need to put [ROT] tag and projectile ended following the target like rocket instead goes missed, also [inaccurate=yes] doesnt seem do much.

so i request feature where bullet can go travel longdistance to target without need to act like rocket, like straight projectile in Phobos 

 
 

### Proposed Code (optional)

[projectiletype]
`isbullettracer=` yes/no  ;shouldnt be tied with [ROT] ,[ARCING] tag so the bullet goes miss and not chase the target
`tracercolor=` r,g,b
`tracerwidth=` 1-100 pixel ;how wide tracer will be draw on screen
`tracerheight=` 1-100 pixel ;how tall tracer will be draw on screen
`tracerlength=` 1-100 pixel ;how length tracer would be draw on screen
`tracerspeed=` 0-100 ;how fast tracer can travel

### Additional Context
This Straight trajectory/tracer on phobos 
![Image](https://github.com/user-attachments/assets/5701a4fe-c029-4fa2-b94b-1a69fb4c808d)

**Comment by Metadorius** [comment]
this is actually not any special tracer logic but just Laser Trails logic that can be applied to any techno or bullet (probably better to have it at an object level)

**Reply by Starkku** [reply]
Original `LineTrail` is `ObjectClass` level thing so there's definitely a precedent for that.
